install_config/aggregate_logging.adoc

```diff
@@ -441,7 +441,10 @@ openshift_logging_es_pvc_dynamic value.
 |`openshift_logging_es_pvc_size`
 |Size of the persistent volume claim to
 create per Elasticsearch instance. For example, 100G. If omitted, no PVCs are
-created and ephemeral volumes are used instead.
+created and ephemeral volumes are used instead. Also specify `openshift_logging_elasticsearch_storage_type=pvc`.
```

Comment:
This isn't necessary to specify, the ansible role infers type=pvc if the storage size is specified and > 0
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_logging/tasks/install_logging.yaml#L83

Reply:
@ewolinetz The original ask in the BZ was to add this line. The BZ is mistaken?
https://bugzilla.redhat.com/show_bug.cgi?id=1599830

This line was added in 3.7?

Reply:
I see this line in 3.7 as well, yes. It may have been added after that bz was created though. https://github.com/openshift/openshift-ansible/blob/release-3.7/roles/openshift_logging/tasks/install_logging.yaml#L83